### PR TITLE
Allow free license request from the Enterprise overview page

### PR DESCRIPTION
**Pull request by bernd**

### CODE REVIEW

File: graylog2-web-interface/src/components/enterprise/EnterpriseFreeLicenseForm.jsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    this.clearValues();
+  };
+
+  render() {
+    const { formFields: { firstName, lastName, company, email } } = this.state;
+
+    return (
+      <form onSubmit={this.submitForm}>
+        <Row>
+          <Col md={5}>
```
Comment (kroepke) [inline graylog2-web-interface/src/components/enterprise/EnterpriseFreeLicenseForm.jsx:78]: This leads to a truncated form:
image

Can we make it a bit wider?

File: graylog2-web-interface/src/pages/EnterprisePage.jsx [hunk 2]
```diff
@@ -1,48 +1,153 @@
...
-          <GraylogClusterOverview />
-          <PluginList />
+          <GraylogClusterOverview layout="compact">
+            <PluginList />
+          </GraylogClusterOverview>
+          <IfPermitted permissions="freelicenses:create">
+            <Row className="content">
+              <Col md={6}>
+                <BiggerFontSize>
+                  <p>Extend Graylog’s Open Source capabilities with a free trial of Graylog Enterprise (up to 5 GB/Day).
```
Comment (kroepke) [inline graylog2-web-interface/src/pages/EnterprisePage.jsx:111]: I would add a break after the first sentence to make it stand out a little more.

File: graylog2-web-interface/src/pages/EnterprisePage.jsx [hunk 3]
```diff
@@ -1,48 +1,153 @@
...
+                    Graylog Enterprise introduces productivity and compliance features designed to help organizations
+                    reduce risk while encouraging collaboration across a large number of users.</p>
+
+                  <p>Graylog Enterprise includes:</p>
+
+                  <EnterpriseFeatureList>
+                    <li>Automated <Link to={Routes.pluginRoute('SYSTEM_ARCHIVES')}><strong>archiving</strong></Link> and
```
Comment (kroepke) [inline graylog2-web-interface/src/pages/EnterprisePage.jsx:118]: If the enterprise plugin isn't installed the internal links will end up in 404s, which isn't pretty.
Should we instead always link to the website?

File: graylog2-web-interface/src/components/enterprise/EnterpriseFreeLicenseForm.jsx (hunk 1 above)
Comment (bernd) [reply graylog2-web-interface/src/components/enterprise/EnterpriseFreeLicenseForm.jsx:78]: Good catch. Yes! :+1:

File: graylog2-web-interface/src/pages/EnterprisePage.jsx (hunk 3 above)
Comment (bernd) [reply graylog2-web-interface/src/pages/EnterprisePage.jsx:118]: Good catch! :+1:

**Pull request merged**